src/main/java/com/google/devtools/build/lib/query2/TransitionsOutputFormatterCallback.java

```diff
@@ -152,10 +145,12 @@ public void processOutput(Iterable<ConfiguredTarget> partialResult) throws Inter
         BuildOptions fromOptions = config.getOptions();
         List<BuildOptions> toOptions = dep.getTransition().apply(fromOptions);
         String hostConfigurationChecksum = hostConfiguration.checksum();
-        String dependencyName =
-            attributeAndDep.getKey() == DependencyResolver.TOOLCHAIN_DEPENDENCY
-                ? PlatformSemantics.RESOLVED_TOOLCHAINS_ATTR
-                : attributeAndDep.getKey().getAttribute().getName();
+        String dependencyName;
+        if (attributeAndDep.getKey() == DependencyResolver.TOOLCHAIN_DEPENDENCY) {
+          continue;
```

Comment:
So we won't be able to see toolchain dependency transitions anymore?

Reply:
Correct. If you'd prefer, I could just set the attribute name for display to something like "[toolchain]", which isn't any more or less meaningful than "$resolved_toolchains_internal".

Reply:
Yeah I think it would be great to retain this information if possible!

Reply:
Done.